include/nuttx/sched.h

```diff
@@ -402,7 +401,7 @@ struct stackinfo_s
   FAR void *stack_alloc_ptr;             /* Pointer to allocated stack          */
                                          /* Needed to deallocate stack          */
   FAR void *adj_stack_ptr;               /* Adjusted stack_alloc_ptr for HW     */
-                                         /* The initial stack pointer value     */
+  FAR void *tls_info_ptr;                /* Thread local storage information    */
```

Comment:
This change is wrong.  Please remove tls_info_ptr which must always have the same valueas stack_alloc_ptr.

Reply:
Ok, I will remove tls_info_ptr